backend/src/controllers/sampleController.ts

```diff
@@ -0,0 +1,88 @@
+import * as express from "express";
+import * as mongoose from 'mongoose';
+import SampleSchema from "../models/sample.model";
+import SampleService from '../services/sampleService';
+
+export default class SampleController {
+    service: SampleService;
+    constructor(service: SampleService) {
+        this.service = service;
+    }
+
+    getAll = async (
+        req: express.Request,
+        res: express.Response
+    ) => {
+        const allSampleData = await this.service.getAll();
+        return res.status(200).json(allSampleData);
+    };
+
+    getById = async (
+        req: express.Request,
+        res: express.Response
+    ) => {
+        const id = new mongoose.Types.ObjectId(req.params.id);
```

Comment:
Wyrzuca mi błąd, jak próbuję odwołać się do endpointów używających ID
```
{
    "message": "Cannot read property 'ObjectId' of undefined"
}
```
próbuję to poprawić, ale na razie nie znalazłem rozwiązania

Reply:
hej, powinno już działać